manifest/com.nalathethird/Numantics/info.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Numantics",
+  "id": "com.nalathethird.Numantics",
+  "description": "A mod that enhances numerical input handling.",
+  "category": "Technical Tweaks",
+  "sourceLocation": "https://github.com/nalathethird/Numantics",
+  "versions": {
+    "1.0.0": {
+      "releaseUrl": "https://github.com/nalathethird/Numantics/releases/tag/1.0.0",
+      "artifacts": [
+        {
+          "url": "https://github.com/nalathethird/Numantics/releases/download/1.0.0/Numantics.dll",
+          "filename": "Numantics.dll",
+          "sha256": "72b8a65dedc58f5698bbfbab2825d18ecab64d454def326033c798a5ba3211a3"
+        }
+      ]
+    },
+    "1.0.1": {
+      "releaseUrl": "https://github.com/nalathethird/Numantics/releases/tag/1.0.1",
+      "artifacts": [
+        {
+          "url": "https://github.com/nalathethird/Numantics/releases/download/1.0.1/Numantics.dll",
```

Comment:
I'd reconsider the easteregg for '0/0' to not close/leave sessions since that is destructive and not going to be clear to a user doing so by accident. While it is not the default, likely will be some who enable it then forget about it. If you want to still have a similar 'effect' without being directly destructive, you could make the user refocus to their local home